flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala

```diff
@@ -1765,7 +1765,7 @@ object TaskManager {
       override def getValue: Double = {
         try{
           fetchCPULoad.map(_.invoke(ManagementFactory.getOperatingSystemMXBean().
-            asInstanceOf[com.sun.management.OperatingSystemMXBean]).
+            asInstanceOf[java.lang.management.OperatingSystemMXBean]).
```

Comment:
I think the change makes the cast unnecessary because thats exactly the type returned by the method.